src/depthmap.h

```diff
@@ -6,7 +6,7 @@
 class DepthMap {
   private:
     cv::Ptr<cv::StereoSGBM> _stereoSGBM;
-	cv::Ptr<cv::StereoBM> _stereoBM;
+	  cv::Ptr<cv::StereoBM> _stereoBM;
```

Comment:
Since this is a cleanup should all tabs be replaced with 2 spaces? This can be the last commit.